_posts/en/newsletters/2021-08-04-newsletter.md

```diff
@@ -0,0 +1,85 @@
+
+- [C-Lightning 0.10.1rc1][C-Lightning 0.10.1] is a release candidate for
+  an upgrade that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  support for [bech32m address encoding][topic bech32], and enhanced support for
+  hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #22006][] adds [documentation][tracing doc] for User-Space, Statically Defined
+  Tracing (USDT) and the first three tracepoints - build support and macros for which were added in
+[Bitcoin Core #19866][]. Users that build Bitcoin Core with eBPF tracing enabled can hook into the
+tracepoints with the provided [example scripts][contrib tracing doc] or write their own tracing
+scripts for greater observability into the node when a new block is connected, inbound P2P messages
+are received, and outbound P2P messages are sent. The documentation also includes usage examples and
+guidelines for the addition of new tracepoints.
+
+- [Eclair #1893][] Different default relay fees for announced and unannounced channels FIXME:bitschmidty
+
+- [Rust-Lightning #967][] adds support for making [keysend-style spontaneous
+  payments][keysend onion] via the
+  `send_spontaneous_payment` function call. With this change, all four LN
+  implementations we cover will have support for keysend.
+
+  The author has also submitted [corresponding documentation][BOLTs #892] (yet
+  unmerged) on keysend payments as a [BLIP][news156 blips] (Bitcoin Lightning
+  Improvement Proposals), a proposed way to document features and best practices
+  which do not belong as part of the spec.
+
+<!-- FIXME:harding to update releases/RCs and topic links -->
+{% include references.md %}
+{% include linkers/issues.md issues="22006,19866,1893,967,892" %}
+[C-Lightning 0.10.1]: https://github.com/ElementsProject/lightning/releases/tag/v0.10.1rc1
+[bitcoin core 22.0]: https://bitcoincore.org/bin/bitcoin-core-22.0/
+[tracing doc]: https://github.com/bitcoin/bitcoin/blob/master/doc/tracing.md
+[contrib tracing doc]: https://github.com/bitcoin/bitcoin/blob/master/contrib/tracing
```

Comment:
I have a minor preference to linking to a specific commit (https://github.com/bitcoin/bitcoin/blob/84ace9aef116a05e034730f2bb2f109d1d77aac7/doc/tracing.md), which should never change, rather than master, which could change or disappear.